flycheck.el

```diff
@@ -7518,6 +7513,14 @@ See URL `https://github.com/eslint/eslint'."
                                    (flycheck-error-message err))))
                           (flycheck-sanitize-errors errors))
                   errors)
+  :enabled (lambda (checker)
+             (let* ((directory (concat
+                                (flycheck-compute-working-directory checker)
+                                "."))
```

Comment:
Adds dot after directory name. Without that dot, ESLint will not look in that directory and ancestors but only ancestors. See sentence before last one in error message:

```
[1085-eslint-use-enabled]~/Projects/elisp/flycheck-upstream$ eslint --print-config test/resources/language/javascript/.
{
  "globals": {},
  "env": {},
  "rules": {
    "no-unused-vars": 1
  },
  "parserOptions": {}
}
[1085-eslint-use-enabled]~/Projects/elisp/flycheck-upstream$ eslint --print-config test/resources/language/javascript/

Oops! Something went wrong! :(

ESLint couldn't find a configuration file. To set up a configuration file for this project, please run:

    eslint --init

ESLint looked for configuration files in test/resources/language and its ancestors.

If you think you already have a configuration file or if you need more help, please stop by the ESLint chat room: https://gitter.im/eslint/eslint
```